enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,256 @@
+
+### Topology Considerations
+TBD
+#### Hypershift / Hosted Control Planes
+TBD
+#### Standalone Clusters
+TBD
+#### Single-node Deployments or MicroShift
+TBD
+### Workflow Description
+TBD
+
+## Design Details
+
+A `SideCar` container is being added within each etcd pod in Openshift cluster in order to take backups without any provided configuration from user's perspective.
+
+As the `SideCar` is being deployed alongside each etcd cluster member, it is possible to keep backups across all master nodes.
+
+Since this work will be enabled with no configuration, it is possible to use define a default values for the `Schedule`, `Retention` independently.
+
+This work utilizes `hostPath` approach, since the `SideCar` container has access to the etcd pod's file system.
+
+Before deciding upon this approach, several approaches that have been investigated. See [Alternatives](#Alternatives).
+
+### Open Questions
+
+* How to distinguish between `NoConfig` backups and configs that are triggered using `EtcdBackup` CR.
+  * The `NoConfig` backups relies on `EtcdBackup` CR with `default` name.
+  * The cluster-etcd-operator reacts to the `default` CR by deploying the backup sidecar containers alongside each etcd member. 
+
+
+* Upon enabling the `AutomatedBackup` feature gate, which approach should be used and according to what criteria.
+  * As the `Noconfig` backups is orthogonal to the [automated backup of etcd](https://docs.openshift.com/container-platform/4.15/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#creating-automated-etcd-backups_backup-etcd), it has been decided to use the same feature gate.
+  
+
+* How should the current backup controller distinguish between both approaches.
+  * The `NoConfig` backups relies on `EtcdBackup` CR with `default` name.
+  
+
+* How to disable the `NoConfig` behaviour, if a user want to.
+  * This has not been decided or implemented yet. 
+
+### Implementation Details/Notes/Constraints
+
+The `NoConfig` backups builds atop of the existing [automated backup of etcd](https://docs.openshift.com/container-platform/4.15/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#creating-automated-etcd-backups_backup-etcd).
+
+
+The `PeriodicBackupController` within `cluster-etcd-operator` checks for `EtcdBackup` CR with `default` name. Once found, a sidecar container is deployed within each etcd pod in Openshift cluster.
+The container runs a backup server with the default schedule as defined in default `EtcdBackup` CR. 
+
+The backups are being stored on the local disk of each master node independently, using `hostPath` volume. This replication is required for disaster recovery scenarios where more than one of the master nodes has been lost.
+
+
+A default `EtcdBackup` CR is used for the `NoConfig` backups. A default schedule of daily at midnight has been chosen. See below
```

Comment:
Midnight UTC could be the middle of peak operational hours for a cluster deployed in APAC or NASA. Is it advisable to take a backup when you're at the peak?

Are backups disruptive to reads/writes at all?

You'll likely need a way for this to be configured or changed